.github/workflows/deploy.yml

```diff
@@ -21,11 +21,13 @@ jobs:
 
     runs-on: [self-hosted, deployment-runner]
     container:
-      image: git.dbyte.xyz/distro/levant
+      image: hashicorp/levant
+      options: --user root
     if: github.ref != 'refs/heads/main' && github.event_name == 'push'
     timeout-minutes: 60
     steps:
       - uses: actions/checkout@v4
+      - run: whoami
```

Comment:
Is this line necessary?

Reply:
definitely not, left that in from debugging 😅